ansible_image_validation/validation-playbooks/nvme_driver_validation.yml

```diff
@@ -0,0 +1,14 @@
+###
+# This Playbook checks if the nvme and pci drivers are present in the given image
+# Author: Navpreet Kaur
+###
+---
+- name: Check if the nvme and pci drivers are present i all images
```

Comment:
Firstly, this playbook does not have an entry point. Refer to https://github.com/Azure/linux-image-validations/blob/master/ansible_image_validation/validation-playbooks/per-vm-validation.yaml

Also, you don't need to add a new playbook as it is a relatively single task. You can add this step in the https://github.com/Azure/linux-image-validations/blob/master/ansible_image_validation/validation-playbooks/per-vm-validation.yaml

Reply:
+1